main.tf

```diff
@@ -138,7 +138,7 @@ module "gke_app" {
   host                       = local.url
   bucket                     = "gs://${local.bucket}"
   bucket_queue               = local.bucket_queue
-  database_connection_string = "mysql://${module.database.connection_string}"
+  database_connection_string = "cloudsql://${module.database.connection_string}"
```

Comment:
Is this tested? I have a task for this test but I didn't finish it yet, but if you did I will archive the task.

Reply:
Its not tested

Reply:
I will put this in front of other tasks and get it tested with the `0.30.0`

Reply:
This won't work.  The connection_string is using the private IP address.  We need to modify the string to use the connection name with colons replaced with period.

Reply:
We can do it in two steps:
1. put it behind a variable, default that variable to off as the default 
2. next release of this module, default to on and customers still have the ability to keep it off until they upgrade the server